jetty-core/jetty-quic/jetty-quic-common/src/main/java/org/eclipse/jetty/quic/common/QuicConnection.java

```diff
@@ -372,17 +372,9 @@ protected Action process()
         }
 
         @Override
-        public void succeeded()
+        protected void onSuccess()
```

Comment:
I'm wondering why we protect the `entry` field with a lock in `process` but not here.   Both methods are called from the same serialized processing, so both should have same locking and memory barriers.

Reply:
Ah is it that the lock is protecting the queue and not the entry field?  If so, some comments would be good.... or just use a concurrent queue.